src/main/kotlin/systems/ajax/englishstudytelegrambot/entity/Entity.kt

```diff
@@ -6,25 +6,22 @@ import org.springframework.data.mongodb.core.mapping.Document
 
 @Document(value = "users")
 data class User(
-    @Id val telegramId: String,
-    val username: String,
-    val libraries: List<Library>
+    @Id val telegramUserId: String
 )
 
 @Document(value = "libraries")
 data class Library(
-    @Id val id: String,
+    @Id val id: String = ObjectId().toHexString(),
```

Comment:
is there a reason to not use ObjectId as a type for id?
Also, don't generate id in your code, delegate this job to database